rest/api_test.go

```diff
@@ -2601,6 +2601,115 @@ func TestBulkGetRevPruning(t *testing.T) {
 
 }
 
+
+
+// Reproduces panic seen in https://github.com/couchbase/sync_gateway/issues/2528
+func TestBulkGetBadAttachmentReproIssue2528(t *testing.T) {
+
+	var rt RestTester
+	var body db.Body
+
+	// Disable rev cache so that the _bulk_get request is forced to go back to the bucket to load the doc
+	// rather than loading it from the (stale) rev cache.  Required when testing against a walrus bucket.
```

Comment:
It might be helpful to expand this comment to explain that the rev cache will be stale because we're planning to directly update the doc in the bucket.

I'm not sure about the 'Required when testing against a walrus bucket' comment - what's the difference when running against a Couchbase bucket?